.gitignore

```diff
@@ -3,6 +3,7 @@
 __pycache__/
 
 *.jpg
+*.txt
```

Comment:
Can you explain why added *.txt in gitignore?

Reply:
I add *.txt in gitignore file because just in case when I download sample datasets not in the data directory, annotation files would be recognized by git.

Reply:
But, there is requirements.txt in root directory. This will affect on that later.
I think this line should be removed.

Reply:
Sorry for checking your comment late. 
Yes, You are correct! I think so too. Thanks for your kind suggestion :)